File: fishing-booker/src/main/java/com/ftn/fishingbooker/model/Address.java
package com.ftn.fishingbooker.model;

import lombok.Data;

import javax.persistence.*;


@Data
@Entity
public class Address {
    @Id
    @Column(name = "id", nullable = false)
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    private String country;

    private String city;

    private String street;

    private Long zipCode;

}

File: IsaClient/src/app/interfaces/address.ts
export interface IAddress{
    id: number ,
    country: string,
    city: string,
    street: string,
    zipCode: number,
}
export interface IAddressMap{
    id: number ,
    country: string,
    city: string,
    street: string,
    zipCode: number,
    latitude: number,
    longitude: number,
}

export class IAddressClass implements IAddressMap{
    id: number = 0;
    country: string = '';
    city: string    = '';
    street: string  = '';
    zipCode: number = 0;
    longitude: number   = 19.833549;
    latitude: number    = 45.267136;
}